tests/sf_service_config_ut/sf_service_config_ut_helpers.h

```diff
@@ -156,6 +156,7 @@
             result = MU_FAILURE; \
             ASSERT_FAIL("Unexpected missing parameter_name"); \
         } \
+        else if (0) {} /* This is here to fool the compiler that there would be another branch that would not end up in an assert fail - without this, if no 'type' parameters exist the compiler thinks the return statement cannot be reached */ \
```

Comment:
>else if (0) {} [](http://example.com/codeflow?start=8&length=14)

This looks like it was fun to figure out.

Reply:
Yep it was :-)